artifacts/operator-configs/controller-conf.yaml

```diff
@@ -55,14 +55,47 @@ data:
   apim-analytics-deployment-readinessProbe-failureThreshold: "3"
   apim-analytics-deployment-env-jvm-heap-memory-xms: "2048m"
   apim-analytics-deployment-env-jvm-heap-memory-xmx: "2048m"
+  apim-analytics-deployment-securityContext-runAsUser: "802"
 
   # pattern specific configs for apim-instances
   p1-apim-deployment-image: "wso2/wso2am:3.2.0"
-  p1-apim-deployment-resources-requests-cpu: "2000m"
-  p1-apim-deployment-resources-requests-memory: "2Gi"
+  p1-apim-deployment-resources-requests-cpu: "1000m"
+  p1-apim-deployment-resources-requests-memory: "1Gi"
   p1-apim-deployment-resources-limits-cpu: "2000m"
   p1-apim-deployment-resources-limits-memory: "2Gi"
 
```

Comment:
Let's use some of the following as common values.

1. Images APIM and Analytics
2. Cpu and memory values
3. JVM heap values
4. Liveness and readiness probes